hack/generate-refdocs.sh

```diff
@@ -1,5 +1,9 @@
 #! /usr/bin/env bash
 
+# Force GOPATH mode so that we don't add the generator as a Contour
+# module dependency.
+export GO111MODULE=off
```

Comment:
This by itself doesn't work for me because it disables Go modules for the `go build` invocation in `gendoc::build`.

Prefixing the `go get` line with `GO111MODULE=off` seems to work, although the `go get` spits an error.

Reply:
see: https://github.com/projectcontour/contour/pull/2006/files#diff-be4c0416b9c2099f206b82ed059fca7bR11